docs/get-started/roles-and-permissions/lbac.md

```diff
@@ -0,0 +1,35 @@
+# Labels for access control in PMM
+
+
+Label-based access control in PMM allows you to manage who has access to metrics based on labels. By creating roles, you can specify which data can be queried based on specific label criteria, for instance, allowing the QA team to view data related to test environments. 
+ 
+With Label-based access control, you can associate multiple labels with a role, ensuring only data from series that match your defined labels is returned. 
+
+
+## Predefined vs Custom labels
+
+PMM supports predefined as well as custom labels. PMM always assigns predefined labels. When an object (Node, Service, or Agent) is created, a user can set initial values.  Predefined labels are automatically assigned and updated by PMM. Custom labels are assigned and updated only by a user.
```

Comment:
What do you have in mind with "predefined labels are automatically updated by PMM"?

Reply:
Its as per the doc: https://confluence.percona.com/pages/viewpage.action?spaceKey=PMM&title=Unified+labels+across+Query+Analytics+and+Prometheus

Reply:
Yeah, we only assign predefined labels, we don't automatically update them.
BTW some of the predefined labels should be assigned by users.

Reply:
@BupycHuk Does this statement hold true?

Reply:
no, I would suggest to remove `and updated` part before `by PMM`

Reply:
ok